modules/ROOT/pages/microsoft-sharepoint-2013-connector.adoc

```diff
@@ -339,15 +385,15 @@ fileContentStream-ref="#[payload]" overwrite="true"
 doc:name="Sharepoint 2013"/>
 ----
 
-In order to upload large files you need to configure your Sharepoint and IIS servers:
+To upload large files you need to configure your Sharepoint and IIS servers:
 
 - Set 'Maximum Upload Size' to 2047MB (max) at SP management console for site.
 - Set connection timeout for IIS site to high value.
 - Set the 'Maximum Allowed Content Length' to 2147483647 for IIS app (at request filtering).
 
-NOTE: The Sharepoint REST API (which the connector uses) supports uploading files up to 2GB. When working with large files it's recommended to provide the system local path to the file (_localFilePath_ parameter's value) as it's the most efficient way to upload it through the connector.
+The Sharepoint REST API (which the connector uses) supports uploading files up to 2GB. When working with large files it's recommended to provide the system local path to the file (localFilePath parameter's value) as it's the most efficient way to upload it through the connector.
```

Comment:
```suggestion
The Sharepoint REST API (which the connector uses) supports uploading files of up to 2 GB. When working with large files, it's best practice to provide the system local path to the file (localFilePath parameter's value) because it's the most efficient way to upload the file through the connector.
```